.github/workflows/README.md

```diff
@@ -85,4 +85,38 @@ jobs:
         uses: sportsball-ai/actions-hub/git-version-tag@<VERSION>
         with:
           version: ${{ needs.reusable_workflow_job.outputs.version }}
-```
\ No newline at end of file
+```
+
+# publish-to-confluence
+
+A configurable github action reusable workflow that will publish specified documents to Confluence.
+Outputs the confluence url of the published document(s) as an environment variable called $confluence_url.
+See https://github.com/justmiles/go-markdown2confluence for more details.
+
+## Example Usage
+
+```yaml
+name: publish README to confluence
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
+
+jobs:
+  publish_to_confluence:
+    if: github.event.pull_request.merged == true
+    uses: sportsball-ai/actions-hub/.github/workflows/publish-to-confluence.yml@v2
+    secrets: inherit
+    with:
+      comment: "this is version 1" # add an optional comment to the published page
+      exclude: ".*temp.md" # List of exclude file patterns (regex) that will be applied on markdown file paths.
+      modified_since: 10 # Only upload files that have modifed in the past n minutes.
```

Comment:
I'm not quite sure I understand the usefulness of this option.

Reply:
modified since last update time on confluence perhaps?

Reply:
docs say "This is particurlarly useful for cron jobs/recurring one-way syncs."  I think the idea is that it compares the update timestamp on the file and only uploads the file if the timestamp is within the specified window of minutes.  I think the idea is that in a release situation, you can tailor the pushing of documents to only those that were just updated as part of the release.

Reply:
That makes sense, thank you!